projects/admin/src/app/record/detail-view/item-detail-view/item-transactions/item-transactions.component.ts

```diff
@@ -14,7 +14,7 @@
  * You should have received a copy of the GNU Affero General Public License
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
-import { Component, OnInit, Input } from '@angular/core';
+import { Component, OnInit, Input, ɵConsole } from '@angular/core';
```

Comment:
What kind of functionnality does it activate?

Reply:
Indeed